gradle-module/src/main/java/org/example/Main.java

```diff
@@ -22,4 +23,11 @@ public static void main(String[] args) {
     private static void test() {
         System.out.println("test");
     }
-}
\ No newline at end of file
+
+   /* asd
+    we
+    asd
+
+    */
+    
+}
```

Comment:
[Category: Consistency] [Severity: Low] The lack of a newline at the end of the file can lead to issues in some tools and environments. It is a good practice to ensure files end with a newline. Rationale: Following this convention can prevent potential issues with version control systems and improve compatibility.